pkg/antctl/raw/multicluster/deploy/member_cluster.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2022 Antrea Authors
```

Comment:
Please check comments in leader_cluster.go

Reply:
Done. Made the same change as leaderr_cluster.go